JS/Lesson34/gulcin/recipe-app/src/modules/recipes/RecipeDetails/index.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react"
+import { Ingredients } from "./ingredients";
+import './styles.css';
+
+export const RecipeDetails = ({ id }) => {
+    const [recipe, setRecipe] = useState([]);
```

Comment:
Since you only need one recipe here, it's better to have the state as object (if there's recipe data) and undefined/null if there's no recipe data found. The initial state would be undefined.
```
    const [recipe, setRecipe] = useState(undefined);
```

Reply:
should we just do like: 
<div>{recipe.title}</div>
 taking the values directly from object?

Reply:
Hep array olmalı gibi düşünmüştüm

Reply:
Yes, you'd later use `{recipe.title}` to get the title. State can be in stored in different types supported by JS (e.g. string, number, boolean, array, object, null, unefined, etc.)

Reply:
done.